community-membership.md

```diff
@@ -40,6 +44,28 @@ reviewing issues/PRs or participate in community discussions on slack/mailing li
 - Guide new contributors to relevant docs/files
 - Help/Motivate new members in contributing to KubeEdge
 
+## Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of a subproject. 
+They are knowledgeable about both the codebase and software engineering principles.
+
+
+### Requirements
+
+- member for at least 1 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- May either self-nominate, be nominated by an approver
+
+
+### Responsibilities and privileges
+
+- Code reviewer status may be a precondition to accepting large code contributions
+- Responsible for project quality control
+- Focus on code quality and correctness, including testing and factoring
```

Comment:
refactoring?  not sure whether this is a typo or not

Reply:
Done.